code/datums/uplink/announcements.dm

```diff
@@ -14,6 +14,7 @@
 	name = "Shipwide Announcement"
 	item_cost = 2
 	desc = "Broadcasts a message anonymously to the entire vessel. Triggers immediately after supplying additional data."
+	antag_roles |= ROLE_CARRION
```

Comment:
I really don't think this is how you do it

Reply:
Fixed.